### PR TITLE
Adapt tests to run on a Jenkins test node

### PR DESCRIPTION
**Pull request by berland**
Copied setup from subscript repository

### CODE REVIEW

**Comment by berland** [comment]
See https://github.com/equinor/subscript/commit/36992871aa0e419c9a42556ccbf8034e53cd1aa9

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import pytest
+
+import ecl2df
+
+
+@pytest.fixture
+def path_to_ecl2df():
+    """Path to installed ecl2df module"""
+    return Path(ecl2df.__file__).parent
```
Comment (xjules) [inline tests/conftest.py:11]: Should we consider absolute path?

Comment (berland) [reply tests/conftest.py:11]: `__file__` is (almost) always absolute, since Python 3.4 it seems.

**Review by sondreso — APPROVED** [review]
LGTM!

**Pull request merged**